### PR TITLE
TESTS-114: As gsy-e-sdk I like to add an integration test that will check sending forecasts is reflected in the results

### PR DESCRIPTION
**Pull request by ERafaelMartinez**
Add gsy-e setup file.

### CODE REVIEW

File: src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+from gsy_e.models.area import Market, Asset
+from gsy_e.models.strategy.external_strategies.load import LoadForecastExternalStrategy
+from gsy_e.models.strategy.external_strategies.pv import PVForecastExternalStrategy
+from gsy_e.models.strategy.load_hours import LoadHoursStrategy
+from gsy_e.models.strategy.storage import StorageStrategy
+
+current_dir = os.path.dirname(__file__)
+print(current_dir)
+print(platform.python_implementation())
```
Comment (Hassan754) [inline src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py:15]: Is there an intended reason for these to reside here?

File: src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def get_setup(config):
+    ConstSettings.GeneralSettings.DEFAULT_UPDATE_INTERVAL = 1
+    ConstSettings.MASettings.MARKET_TYPE = 2
```
Comment (Hassan754) [inline src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py:20]: Nit
```suggestion
    ConstSettings.MASettings.MARKET_TYPE = SpotMarketTypeEnum.TWO_SIDED.value
```

**Review by Hassan754 — COMMENTED** [review]
General comment, now that we have a dedicated testing project, I suggest moving (or at least starting to put) the setup files needed for tests there

File: src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py (hunk 1 above)
Comment (ERafaelMartinez) [reply src/gsy_e/setup/strategy_tests/external_devices_send_forecasts.py:15]: Actually not. I copied an already existing strategy and changed what I needed, so I think is just a copy paste thing. I've removed it anyway. Thanks

**Comment by ERafaelMartinez** [comment]
Adapted to comments. Appreciate your re-review. Thanks

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Review by Hassan754 — APPROVED** [review]

**Pull request merged**